ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -39,6 +39,5 @@ tests:
     ./hack/ci/test.sh
   openshift_installer_custom_test_image:
     cluster_profile: gcp
-    from: pipeline:kubevirt-test
+    from: stable:kubevirt-test
```

Comment:
Can you explain why this needs to be done, please? I always thought that during the run the test image would be built as part of the pipeline and therefore it should be referenced with pipeline? Don't we get another image otherwise?